HW2/P3/P3.txt

```diff
@@ -0,0 +1,25 @@
+As shown below, the performance of implementing AVX yields much better time efficiency. And the more threads, the faster.
+1) nonAVX -> AVX_1: 706%
+2) AVX_1 -> AVX_2: 208%
+3) AVX_2 -> AVX_4: 142%
+4) AVX_4 -> AVX_8: 110%
```

Comment:
I'm curious what machine you ran this on since you got more pronounced increases than I did. I'm using an Ubuntu VM.